core-services/prow/02_config/_config.yaml

```diff
@@ -78,6 +78,7 @@ deck:
         name: html
       required_files:
       - .*/e2e-intervals.*\.html
+      - .*/event-filter\.html
```

Comment:
/hold

Reply:
I am very sure this is not what you want, now you require both of them. You want to OR them, right? If so, it must be one regex

Reply:
ack, thanks @alvaroaleman